jkube-kit/config/image/src/test/java/org/eclipse/jkube/kit/config/image/ImageConfigurationTest.java

```diff
@@ -27,12 +27,13 @@ class ImageConfigurationTest {
     @Test
     void testBuilder() {
         // Given
-        BuildConfiguration mockJKubeBuildConfiguration = mock(BuildConfiguration.class);
-        when(mockJKubeBuildConfiguration.getUser()).thenReturn("super-user");
```

Comment:
Could you please remove import statements for mockito now that they are not used?

Reply:
Just pushed a commit of that, review it once you get time.